src/schema.js

```diff
@@ -92,6 +92,7 @@ class Schema {
 
     this.generateDefaultApi(typeDef, def); // Generate default
     this.mixinType(typeKey, typeDef); // Mixin default type
+    this.mixinNamedParamFacade(typeDef, def);
```

Comment:
I know this is not your doing, but the naming `typeDef` and `def` has always confused me, perhaps (`api`, `qixStruct`) or something like that would be clearer when reading the code below